src/tools/ceph_kvstore_tool.cc

```diff
@@ -64,7 +70,18 @@ class StoreTool
 	exit(1);
       }
     }
-    db.reset(db_ptr);
+    db = db_ptr;
+  }
+
+  ~StoreTool() {
+    if (bluestore) {
+      assert(0 == bluestore->umount());   
```

Comment:
please move the call with side effects out of the assert, in case we compile out assert() in the future.

Reply:
thank you Sage. done, remove assert here.